clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -173,7 +173,7 @@ public class ConsumerConfig extends AbstractConfig {
             "(e.g. because that data has been deleted): " +
             "<ul><li>earliest: automatically reset the offset to the earliest offset" +
```

Comment:
Strictly speaking, you're missing an `</li>` at the end of this line.

Reply:
Yes, you're right 😄 
Just add it.